graph/pull_v2.go

```diff
@@ -76,6 +76,9 @@ func (p *v2Puller) pullV2Repository(tag string) (err error) {
 	broadcaster.Add(p.config.OutStream)
 	if found {
 		// Another pull of the same repository is already taking place; just wait for it to finish
+		if dryRun {
+			broadcaster.Write(p.sf.FormatStatus(tag, "!!! Another pull of the same image is already running, dry-run is not possible unless you restart the Docker daemon !!!"))
```

Comment:
Thanks for your comments.
Our client requesting a pull in "dry-mode" shouldn't affect other pulls.
Will make changes as soon as I can.